landbosse/excelio/XlsxManagerRunner.py

```diff
@@ -87,7 +81,7 @@ def extract_module_type_operation_lists(self, runs_dict):
                     result.extend(value)
         return result
 
-    def extract_csv_lists(self, runs_dict):
+    def extract_details_lists(self, runs_dict):
```

Comment:
Renamed so that it is more clear that this method extracts data for the `detail` tab.